caffe2/core/tensor.h

```diff
@@ -222,17 +222,18 @@ class CAFFE2_API TensorImpl : public c10::intrusive_ptr_target {
     if (size() > 0) {
       if (storage_.dtype().copy()) {
         CAFFE_ENFORCE(
-            GetDeviceType() == CPU,
+            GetDeviceType() == DeviceType::CPU,
```

Comment:
You didn't end up adding an `constexpr auto CPU = DeviceType::CPU` somewhere global in the Caffe2 namespace? (It's OK either way, just a little surprised!)

Reply:
oh we have that, this should be a change I did before I define that.